science/sollya/Portfile

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem      1.0
+
+name            sollya
+version         8.0
+revision        0
+
+categories      science
+license         CeCILL-C
+maintainers     {langly.org:kenneth.ostby @langly} openmaintainer
+description     Sollya is both a tool environment and a library for safe floating-point code development
+long_description \
+                Sollya is both a tool environment and a library for safe floating-point code \
+                development. It is particularily targeted to the automatized implementation of \
+                mathematical floating-point libraries (libm). Amongst other features, it offers \
+                a certified infinity (supremum) norm and a fast Remez algorithm.
+
+homepage        https://www.sollya.org/
+master_sites    https://www.sollya.org/releases/sollya-${version}/
+
+checksums       rmd160  4da6a6c5181f21cfcef795565bb8183b86fb7aef \
+                sha256  58d734f9a2fc8e6733c11f96d2df9ab25bef24d71c401230e29f0a1339a81192 \
+                size    4425491
+
+compiler.cxx_standard 2011
+
+depends_lib-append \
+                port:mpfr \
+                port:gmp \
+                port:mpfi \
+                port:fplll
```

Comment:
when you check the resulting libraries you'll notice that it links to the `libxml2` and `zlib` but the files located in `/usr/local`. It would be better to add them as dependencies here so that the MacPorts' provides ones will be used.

In other words:
```
+                port:libxml2 \
+                port:zlib \
```